tests/erlang_tests/test_monitor.erl

```diff
@@ -34,6 +34,10 @@ start() ->
     ok = test_monitor_demonitor_from_other(),
     ok = test_monitor_registered(),
     ok = test_monitor_registered_noproc(),
+    ok = test_alias(),
+    ok = test_monitor_alias(),
+    ok = test_monitor_alias_explicit_unalias(),
+    ok = test_monitor_down_alias(),
```

Comment:
We should also test unhappy paths, such as:
1. double/triple unalias
2. alias a dead process
3. unalias a dead process

In addition multiple aliases should be tested,  this is legit but is should be properly tested.